File: ExampleRealm/app/src/main/java/com/pratamawijaya/examplerealmio/MainActivity.java
package com.pratamawijaya.examplerealmio;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.ActionBarActivity;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Button;

import butterknife.ButterKnife;
import butterknife.InjectView;
import butterknife.OnClick;
import io.realm.Realm;


public class MainActivity extends ActionBarActivity {

    @InjectView(R.id.btnRead)
    Button btnRead;
    @InjectView(R.id.btnInsert)
    Button btnInsert;

    private Realm realm;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.inject(this);
    }

    @OnClick(R.id.btnInsert)
    public void insertData() {
        startActivity(new Intent(this, InsertDataActivity.class));
    }

    @OnClick(R.id.btnRead)
    public void getData() {
        startActivity(new Intent(this, ReadDataActivity.class));
    }


    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.menu_main, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();

        //noinspection SimplifiableIfStatement
        if (id == R.id.action_settings) {
            return true;
        }

        return super.onOptionsItemSelected(item);
    }
}

File: KotlinLanguage/ExampleKotlinAnko/app/src/main/kotlin/com/pratamawijaya/examplekotlinanko/MainActivity.kt
package com.pratamawijaya.examplekotlinanko

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.jetbrains.anko.button
import org.jetbrains.anko.editText
import org.jetbrains.anko.linearLayout
import org.jetbrains.anko.onClick

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        linearLayout {
            val editName = editText {
                button("jogja") {
                    onClick { "Papoy" }
                }
            }
        }

    }
}
